pkg/prometheus/rules.go

```diff
@@ -224,15 +222,14 @@ func makeRulesConfigMaps(p *monitoringv1.Prometheus, ruleFiles map[string]string
 		{},
 	}
 	currBucketIndex := 0
-	sortedNames := sortKeyesOfStringMap(ruleFiles)
 
-	for _, filename := range sortedNames {
+	for filename, filecontent := range ruleFiles {
 		// If rule file doesn't fit into current bucket, create new bucket
-		if bucketSize(buckets[currBucketIndex])+len(ruleFiles[filename]) > maxConfigMapDataSize {
+		if bucketSize(buckets[currBucketIndex])+len(filecontent) > maxConfigMapDataSize {
```

Comment:
I think we need to run gofmt on this file. Is that possible?

Reply:
My editor does so on every save. Does it complain on your version?

Reply:
looks like it's context dependent [0]

[0] https://play.golang.org/p/9ZbTzwNUHU7